kuma/wiki/forms.py

```diff
@@ -776,6 +776,17 @@ def akismet_parameters(self):
         parameters.update(self.akismet_parameter_overrides())
         return parameters
 
+    def akismet_call(self, parameters):
+        """
+        Skip Akismet check if the content is the same.
+
+        This happens if the edit is to a non-content field, such as
+        setting or clearing the technical review flag.
+        """
+        if not parameters.get('comment_content', 'omitted'):
```

Comment:
I think it should be written like
``if not parameters.get('comment_content') == '':``
So its more readable. I get confused by reading the code! 😕